config/src/main/resources/examples/filemgr/cmd-line-actions.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
maybe we should use maven-ant-tasks to copy these in the prepare-resources phase or in the compile phase (whatever you need them for) instead of physically copying the files?